ParquetSharp.Dataset.Test/Filter/TestTimestampFilter.cs

```diff
@@ -45,12 +42,83 @@ public void TestComputeMask(TimeUnit unit)
         }
     }
 
+    [Test]
+    public void TestComputeEqMask([Values] TimeUnit unit)
+    {
+        var value = new DateTime(2025, 5, 19, 12, 34, 0);
+        var filter = Col.Named("timestamp").IsEqualTo(value);
+        TestComputeComparisonMask(unit, filter, dt => dt == value);
+    }
+
+    [Test]
+    public void TestComputeGtMask([Values] TimeUnit unit)
+    {
+        var value = new DateTime(2025, 5, 19, 12, 34, 0);
+        var filter = Col.Named("timestamp").IsGreaterThan(value);
+        TestComputeComparisonMask(unit, filter, dt => dt > value);
+    }
+
+    [Test]
+    public void TestComputeGtEqMask([Values] TimeUnit unit)
+    {
+        var value = new DateTime(2025, 5, 19, 12, 34, 0);
+        var filter = Col.Named("timestamp").IsGreaterThanOrEqual(value);
+        TestComputeComparisonMask(unit, filter, dt => dt >= value);
+    }
+
+    [Test]
+    public void TestComputeLtMask([Values] TimeUnit unit)
+    {
+        var value = new DateTime(2025, 5, 19, 12, 34, 0);
+        var filter = Col.Named("timestamp").IsLessThan(value);
+        TestComputeComparisonMask(unit, filter, dt => dt < value);
+    }
+
+    [Test]
+    public void TestComputeLtEqMask([Values] TimeUnit unit)
+    {
+        var value = new DateTime(2025, 5, 19, 12, 34, 0);
+        var filter = Col.Named("timestamp").IsLessThanOrEqual(value);
+        TestComputeComparisonMask(unit, filter, dt => dt <= value);
+    }
+
+    private static void TestComputeComparisonMask(TimeUnit unit, IFilter filter, Func<DateTime, bool> predicate)
```

Comment:
We have two code paths, with and without nulls. The tests covers only the case with nulls, so we should add the non-null tests too.

Reply:
Good point, I've added tests for the non-null case too, and also done the same for the date filter tests.